sentences/en/_common.yaml

```
@@ -394,8 +394,8 @@ expansion_rules:
   all: "(all[[ of] the]|every[ single]|each[ and every])"
   are_all: "[<are> ]<all>"
   home: "(home|house|appartment|flat)"
-  everywhere: "(everywhere|in (all|each)[[ of] the] room[s]|in every room[s]|in the <home>)"
-  here: "(here|in here|in this room|in the room)"
+  everywhere: "(everywhere|all over|[<in> ](all|each)[[ of] the] room[s]|[<in> ]every room[s]|(in|across|around|throughout) the [(entire|whole) ]<home>|<in> <all> area[s])"
+  here: "(here|[in ]here|[in ](this|the) (room|area|space))"
```

```suggestion
  here: "([in] here|[in] (this|the) (room|area|space))"
```
`(here|[in ]here)` is redundant and we decided to go withe the easily legible stype of `[optional] part` instead of the more optimized `[optional ]part`.